homeassistant/components/ads/light.py

```diff
@@ -21,25 +17,23 @@
     AdsEntity,
 )
 
-DEFAULT_NAME = "ADS Light"
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_ADS_VAR): cv.string,
-        vol.Optional(CONF_ADS_VAR_BRIGHTNESS): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-    }
-)
-
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the light platform for ADS."""
+    entities = []
+
+    if discovery_info is None:  # pragma: no cover
```

Comment:
Why `# pragma: no cover` when there are not tests currently?
```suggestion
    if discovery_info is None:
```